specs/bundling.md

```diff
@@ -0,0 +1,27 @@
+## Safe Module Merkle Tree Bundling Logic
+
+### Goal
+A function that transforms an arbitrary set of transactions on an arbitrary set of networks into a Merkle Tree which is executable by the SphinxModule. See the SphinxModule Merkle Tree spec for more detail on the exact structure and content of the Merkle tree and its leafs.
+
+### Secondary Goals
+- Simple and auditable
+- Should be a function that is agnostic to the source of transactions so it can be used to assemble a Merkle Tree based on transactions generated from any scripting framework (i.e Foundry, Hardhat Ignition, some arbitrary future framework)
+
+### Input
+Accepts an object where the keys are canonical chain ids and the values are deployment data objects.
+
+#### Deployment Object Fields:
+- nonce: The current nonce of the Safe on the target network
+- txs: An array of Sphinx Transaction objects
+
+#### Sphinx Transaction Object Fields:
+- to: The destination address for the transaction
+- value: The amount of native gas token to send from the Safe as part of the transaction
```

Comment:
It's implied, but it'd be nice to explicitly say that the value is being transferred to the destination address, i.e. the `to` field